mllib/src/test/scala/org/apache/spark/mllib/feature/Word2VecSuite.scala

```diff
@@ -38,6 +38,13 @@ class Word2VecSuite extends FunSuite with MLlibTestSparkContext {
     assert(syms.length == 2)
     assert(syms(0)._1 == "b")
     assert(syms(1)._1 == "c")
+
+    val word2VecMap = model.getVectors
+    val newModel = new Word2VecModel(word2VecMap)
+    val newSyms = newModel.findSynonyms("a", 2)
```

Comment:
Instead of testing newModel like this, can you just compare the model data with the original model?